### ISSUE TITLE
Unable to start/load a game on my 16:10 screen

### ISSUE THREAD

**Issue by BlueRab** [voxeltycoon/issues#737]
**Describe the bug**
The game loading screen stop at the the last step (progress bar ~95%) when I start the game on my 16:10 screen.
When I load the same game on my 16:9 screen, there is no problem.

**Steps to reproduce**
Start any game with a 16:10 screen. (I run at 1 680 × 1 050 px).

**Build version**
0.85.1 (same bug on 0.85)

**Platform**
Linux Kubuntu (20.04 LTS)

**Reproducibility**
Every time

**Logs/saves**
A fresh new game reproduce the same bug

**Comment by andrewpey** [comment]
Can we see the [log file](https://github.com/voxeltycoon/issues/discussions/650), please?

**Comment by vitorboschi** [comment]
I believe I have the same/related issue, so I'll provide a bit more info on my findings: 
- the problem seems to happen in other situations as well, like opening clicking on a research or tool for the first time in a while
- the game is still running and the cursor changes when moved over buttons. The game still reacts to clicks on them, but nothing is rendered while in this state
- If I alt+tab, then move the game to another screen, it recovers itself and starts rendering again. My guess is that the change of resolution resets the renderer and that makes it recover.

I don't think the problem is related to the screen being 16:10, because both my screens are 16:9 and the problem still happens
log is attached
[Player.log](https://github.com/voxeltycoon/issues/files/6624384/Player.log)

**Comment by andrewpey** [comment]
@vitorboschi from the logs it looks like Steam-on-Linix related issue (something with their overlay rendering?). Can you try to disable the overlay and see if this helps?

**Comment by vitorboschi** [comment]
Checked again with overlay disabled and the problem still happen.

Here's the updated log:
[Player.log](https://github.com/voxeltycoon/issues/files/6626260/Player.log)

I can't check that link now, but will do later and report back. Let me know if there's anything else I can check. As a developer myself, I'm pretty puzzled with this bug

**Comment by vitorboschi** [comment]
**More updates**

**TLDR:** Changing from fullscreen to windowed works around the issue

**Details**
- I don't think the linked issue is related to this one at all, because everything else (games, CUDA accelerated applications, steam itself) is working fine. VoxelTycoon also somewhat works
- I've disconnected one of my screen to check whether this is related to multi-screens setup, and it still happens, so I think it's safe to say that this is not a problem either
- What does help is to disable the fullscreen mode. When windowed I can play normally in any screen
- I've reenabled the Steam overlay as the game does not seem affected by it: the issue still happen in fullscreen mode no matter if the overlay is active or not, and it does not happen in windowed mode, despite of overlay status
- I've checked `dmesg` and made sure nothing was being logged while the game runs

**Comment by andrewpey** [comment]
Thanks for the extra info.
